jwst/datamodels/tests/test_models.py

```diff
@@ -59,6 +60,17 @@ def make_models(tmp_path):
     }
 
 
+def test_init_from_pathlib(tmp_path):
+    """Test initializing model from a PurePath object"""
+    path = tmp_path / "pathlib.fits"
```

Comment:
Is `tmp_path` already a `Path` object? Or something else?

Reply:
I honestly don't know where/how tmp_path gets initialized, but it seems to be an environment where pytest creates fits files during a run, based on the fixture above this test. I believe it should be a string, but I convert it to a Path object in the datamodels.open() call. I notice in the test comment that I say it uses a PurePath object, though I believe it's just a Path object, if that matters. I can clear that up.

Reply:
It actually is a Path object; [see the docs](https://docs.pytest.org/en/6.2.x/tmpdir.html). So you should be able to get away with not actually re-Path-ing it.